examples/powder_averaging.jl

```diff
@@ -75,14 +75,13 @@ fig
 # factor, a list of radius values (Å⁻¹), and a density parameter (Å⁻²) that will
 # control the number of wave vectors to sample at each radius. For each radius
 # `r`, the function will generate wavevectors on a sphere of this radius and
-# retrieve their [`intensities`](@ref). These intensities will be broadened, as
+# retrieve their [`intensities_interpolated`](@ref). These intensities will be broadened, as
 # just demonstrated above, and then averaged to produce a single vector of
 # energy-intensities for each `r`. Note that our `powder_average` function
-# passes most of its keywords through to [`intensities`](@ref), so it can be
-# given `kT`, `formfactors`, etc., and these parameters will be applied to the
-# calculation.
+# passes most of its keywords through to [`intensities_interpolated`](@ref), so it can be
+# given an [`intensity_formula`](@ref).
 
-function powder_average(sf, rs, density; η=0.1, mode=:perp, kwargs...)
+function powder_average(sf, rs, density; η=0.1, kwargs...)
```

Comment:
We should probably just use the built-in function if we're now exporting it. I can update this.